src/Kreta/TaskManager/Application/Organization/RemoveOwnerToOrganizationCommand.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Kreta package.
+ *
+ * (c) Beñat Espiña <benatespina@gmail.com>
+ * (c) Gorka Laucirica <gorka.lauzirika@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Kreta\TaskManager\Application\Organization;
+
+class RemoveOwnerToOrganizationCommand
+{
+    private $removerId;
+    private $userId;
+    private $organizationId;
+
+    public function __construct(string $removerId, string $userId, string $organizationId)
```

Comment:
Maybe te target of the handler (owner to remove) must be first param. WDYT? 
`__construct(string $userId, string $organizationId, string $removerId)`